src/virtualenv/activation/via_template.py

```diff
@@ -36,12 +36,8 @@ def replacements(self, creator, dest_folder):
         current_platform = sysconfig.get_platform()
         platforms = ["mingw", "cygwin", "msys"]
```

Comment:
can you move this variable inline, and make it a tuple? 😊 And end it with a coma to have each name on it's own line to make it merge friendly? Thanks!